src/gui/src/browserWidget.cpp

```diff
@@ -141,8 +141,15 @@ BrowserWidget::BrowserWidget(
 
   display_controls_warning_->setStyleSheet("color: red;");
 
-  model_->setHorizontalHeaderLabels(
-      {"Instance", "Master", "Instances", "Macros", "Modules", "Area"});
+  model_->setHorizontalHeaderLabels({"Instance",
+                                     "Master",
+                                     "Local Instances",
```

Comment:
Keep the most pertinent numbers to the left, most easily visible.

move "Local variants", right of "Area". Most of the time it is the recursive count(including current & childern) number that is interesting.